src/installer/installer.go

```diff
@@ -304,9 +304,12 @@ func (i *installer) downloadHostIgnition() (string, error) {
 	return dest, err
 }
 
-func waitForNetworkType(kc k8s_client.K8SClient) error {
+func (i *installer) waitForNetworkType(kc k8s_client.K8SClient) error {
 	return utils.WaitForPredicate(networkTypeTimeoutSeconds*time.Second, time.Second, func() bool {
 		_, err := kc.GetNetworkType()
+		if err != nil {
+			i.log.WithError(err).Error("Failed to get network type")
```

Comment:
Will it become a spam log? @eranco74 ?
Maybe instead you can add a log before you start the monitoring?

Reply:
This is the reason this PR exist https://coreos.slack.com/archives/C014N2VLTQE/p1610473832056400